mbof/views.py

```diff
@@ -66,9 +68,37 @@ class EventViewSet(OcellusViewSet):
     queryset = Event.objects.all().order_by('postingTime')
     serializer_class = EventSerializer
 
+    def retrieve(self, request, *args, **kwargs):
+        log_action(request)
+        instance = self.get_object()
+        serializer = self.get_serializer(instance)
+        serializer_data = serializer.data
+        if serializer_data['owner'] == self.request.user.username:
+            serializer_data['owner'] = "True"
+        else:
+            serializer_data['owner'] = "False"
+        return Response(serializer_data)
```

Comment:
Why use strings `"True"` and `"False"` rather than the Boolean values `True` and `False`?  Also, why not let the serializer add this field?

How are these values used later?  Does JS in the UI use it to determine whether certain parts of the data are shown to the user?  That's not safe, because a clever user could retrieve the JSON (using only the dev. tools built into the browser) and see all of the data.